app/controllers/application_controller.rb

```diff
@@ -52,6 +53,13 @@ def clear_searches(except: [])
     end
   end
 
+  def allow_cors_requests
+    if Rails.env.development?
```

Comment:
Should this only be in development? Also, maybe we only need to allow it for your GlobalStatsController?

`controller_name.inquiry.global_stats?`

Reply:
Last week I kept running into issues in development because of localhost ports being considered CORS. I think once its in stage/prod we shouldn't run into those problems. Subdomains and nginx route filtering shouldn't run into these issues.

We can easily add this to the site as a global config. Although I think keeping CORS locked down to the minimal required for stage/prod is best. Let me know what you think.

Reply:
ok then lets do this. Remove this if statement and then do

`before_action :allow_cors_requests, unless: -> { Rails.env.production? }`

Reply:
Nice. Clean. Will change this.